src/app/modules/InvitationManagement/components/InvitationProfile/InvitationProfile.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { DynamicFormRenderer, Loading } from '@app/components';
+import { useVerifyTokenQuery } from '@app/models';
+import { useCreateInvitationTokenMutation } from '@app/models';
+import invitationSchema from '@app/modules/InvitationManagement/schema/invitation-profile.json';
+import { Alert, AlertGroup } from '@patternfly/react-core';
+
+const InvitationProfile: React.FC = () => {
+
+    let input;
+    const [createTokenMutateFunction, { data, loading, error }] = useCreateInvitationTokenMutation({
+         onCompleted: (data) => {
+         },
+    });
+
+
+  if (loading) return 'Submitting...';
```

Comment:
These two `if` statements will cause the issue after click on submit button. It will hide the form on page and will display message only. 
We can add alert on top of form and will display inline alert with message and loading icon. It will display when loader will be true.